app/views/users/sessions.py

```diff
@@ -141,6 +146,77 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    main = []
+    data = ["Speaker Name", "Speaker Session(s)", "Speaker Email", "Speaker Status", \
```

Comment:
@LuD1161 I still dont get why we need to do 

```python
main = []
    data = ["Speaker Name", "Speaker Session(s)", "Speaker Email", "Speaker Status", \
            "Speaker Mobile", "Speaker Organisation", "Speaker Position"]
    main.append(data)
``` 

When it could be this. Doesn't this look cleaner ?

```python
main = [
    [
      "Speaker Name", "Speaker Session(s)", "Speaker Email", "Speaker Status",\
      "Speaker Mobile", "Speaker Organisation", "Speaker Position"
    ]
  ]
```